docs/lesiaturuta/5-logic/index.html

```diff
@@ -0,0 +1,127 @@
+                        </li>
+                        <li>
+                            <a class="menu__link" href="../PokemonProject/login.html">Login</a>
+                        </li>
+
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <main class="main calculate justify__content--center">
+            <div class="container justify__content--center flex-direction-column calculate__container">
+                <h1 class="calculate__title">Calculate Pokemons</h1>
+                <div class="field">
+                    <label class="field__label flex-direction-column">
+                        max 10
+                        <input class="calculate__first-value gap-top12" id="firstValue">
+                    </label>
+                    <label class="flex-direction-column justify__content--between">
+                &nbsp;
+                        <select class="calculate__operation gap-top12" name="operation" id="operation">
+                            <option value="+">+</option>
+                            <option value="-">-</option>
+                            <option value="/">/</option>
+                            <option value="*">*</option>
+                            <option value="**">**</option>
+                        </select>
+                    </label>
+                    <label class="flex-direction-column justify__content--between">
+                &nbsp;
+                        <input class="calculate__second-value gap-top12" id="secondValue">
+                    </label>
+                    <div class="flex-direction-column justify__content--between">
+                &nbsp;
+                        <button class="calculate__button gap-top12" id="calculate" type="button">=</button>
+                    </div>
+                    <div class="flex-direction-column justify__content--between">
+                        <span class="field__label">max 100</span>
+                        <span class="calculate__result" id="result"></span>
+                    </div>
+                </div>
+                <p>
+                    <span data-date></span>
+                    <span data-timeMs></span>
+                </p>
+                <div class="calculate__show">
+                    <div class="calculate__show-first" data-firstValue></div>
+                    <div class="calculate__show-operation" data-operation></div>
+                    <div class="calculate__show-second" data-secondValue></div>
+                    <div class="calculate__show-equals" data-equals></div>
+                    <div class="calculate__show-result" data-showImages></div>
+                </div>
+
+            </div>
+        </main>
+        <footer class="footer">
+            <ul class="footer__icons">
+                <li class="footer__icon-facebook item gap-right33">
+                    <a class="item__link-facebook" href="#">
```

Comment:
Please add a link or add a visual representation, that the element is not accessible for interaction (e.g. `cursor: not-allowed;`)

Reply:
fixed